src/libs/getPlaidLinkTokenParameters/types.ts

```diff
@@ -0,0 +1,7 @@
+type PlaidLinkTokenParameters =
+    | { android_package: string } // eslint-disable-line @typescript-eslint/naming-convention
+    | { redirect_uri: string }; // eslint-disable-line @typescript-eslint/naming-convention
```

Comment:
```suggestion
type PlaidLinkTokenParameters = {
    // eslint-disable-next-line @typescript-eslint/naming-convention
    android_package?: string;

    // eslint-disable-next-line @typescript-eslint/naming-convention
    redirect_uri?: string;

    allowDebit?: boolean;
    bankAccountID?: number;
};
```

Since we have a platform file which uses `android_package` and others which uses `redirect_uri`, a better representation of this data would be with these parameters as optional ones.

Also, `getPlaidLinkTokenParameters` is only used in `src/libs/actions/Plaid.js` and it seems that they add `allowDebit` and `bankAccountID` to this params, so we should include them as well.